spec/services/account_reset/create_request_spec.rb

```diff
@@ -23,5 +24,16 @@
         expect(response.to_h[:message_id]).to be_present
       end
     end
+
+    context 'when requesting_issuer is passed' do
+      let(:user) { build(:user) }
+
+      it 'it stores requesting_issuer' do
+        create_request.call
+        arr = AccountResetRequest.find_by(user_id: user.id)
+
+        expect(arr.requesting_issuer).to eq requesting_issuer
```

Comment:
I use `arr` as a shorthand for array, I would consder calling this `request` or `reset_request`
```suggestion
        reset_request = AccountResetRequest.find_by(user_id: user.id)

        expect(reset_request.requesting_issuer).to eq requesting_issuer
```